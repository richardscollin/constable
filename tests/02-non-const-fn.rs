use constable::lookup;

#[lookup]
fn foo(_x: u8) -> bool {
    false
}

fn main() {}
