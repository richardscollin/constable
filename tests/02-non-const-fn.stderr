error: constable: function must be const
 --> tests/02-non-const-fn.rs:3:1
  |
3 | #[lookup]
  | ^^^^^^^^^
  |
  = note: this error originates in the attribute macro `lookup` (in Nightly builds, run with -Z macro-backtrace for more info)
